numpy/doc/indexing.py

```diff
@@ -295,6 +295,13 @@
 
 Likewise, slicing can be combined with broadcasted boolean indices: ::
 
+ >>> b = y > 20
+ >>> b
+ array([[False, False, False, False, False, False, False],
+       [False, False, False, False, False, False, False],
+       [False, False, False, False, False, False, False],
+       [ True,  True,  True,  True,  True,  True,  True],
+       [ True,  True,  True,  True,  True,  True,  True]])
```

Comment:
Let's leave an empty line here. Else LGTM

Reply:
You are likely right (have to double check), OTOH, the whole file seems to not put the blank line. I was wondering a bit if we need to print out `b`, but I guess it is indeed better to do.

The whole file can probably use more fixups, but I will just put this in. Thanks @oribro and also @abhinavsagar review is much appreciated and always in short supply!

Reply:
Any followup PRs welcome of course ;).